packages/cdk/src/config.ts

```diff
@@ -0,0 +1,214 @@
+import { GetParametersByPathCommand, SSMClient } from '@aws-sdk/client-ssm';
+import {
+  ExternalSecret,
+  ExternalSecretPrimitive,
+  ExternalSecretPrimitiveType,
+  MedplumInfraConfig,
+  MedplumSourceInfraConfig,
+  OperationOutcomeError,
+  badRequest,
+  validationError,
+} from '@medplum/core';
+
+const DEFAULT_AWS_REGION = 'us-east-1';
+const VALID_PRIMITIVE_TYPES = ['string', 'boolean', 'number'];
+
+export async function fetchParameterStoreSecret(path: string, key: string): Promise<string> {
+  const region = DEFAULT_AWS_REGION;
+  const client = new SSMClient({ region });
+
+  let nextToken: string | undefined;
+  do {
+    const response = await client.send(
```

Comment:
This will be O(n^2) lookup.

We can either get *all* the parameters for a given path prefix, and then match them up.

Or we can get parameters directly using `GetParametersCommand`:  https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-ssm/classes/getparameterscommand.html